app/policies/transaction_policy.rb

```diff
@@ -1,23 +1,43 @@
 class TransactionPolicy < ApplicationPolicy
+  def show?
+    return true if beis_user?
+    record.parent_activity.organisation == user.organisation
+  end
+
   def create?
-    Pundit.policy!(user, record.parent_activity).create? && associated_report.present?
+    return false if record.parent_activity.level.nil?
```

Comment:
In which cases activities would have no level? Those that are incomplete and the `form_state` is before the level step?

Reply:
You got it, you can hit the create activity button and we go ahead and create a new activity without a level or parent.